cf-agent/verify_files.c

```diff
@@ -617,8 +617,9 @@ static PromiseResult VerifyFilePromise(EvalContext *ctx, char *path, const Promi
     free(chrooted_path);
     if (AttrHasNoAction(&a))
     {
-        Log(LOG_LEVEL_WARNING,
-            "No action was requested for file '%s'. Maybe a typo in the policy?", path);
+        Log(LOG_LEVEL_VERBOSE, "No action was requested for file '%s'. "
+            "Maybe all attributes are skipped due to unresolved arguments in policy functions? "
+            "Maybe a typo in the policy?", path);
```

Comment:
I'd keep this in place. People can still write `files` promises with no attributes maybe thinking the files would be created or something like that.

Reply:
@larsewi @vpodzime maybe it makes sense to move the detection / warning for that to another place? i.e. this can be checked during parsing.

Reply:
Good idea. And maybe this is even a generic thing we can do for all promises? Perhaps with one exception or two?